src/dyad/modules/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+set(DYAD_MODULE_SRC ${CMAKE_CURRENT_SOURCE_DIR}/dyad.c)
+set(DYAD_MODULE_PRIVATE_HEADERS )
+set(DYAD_MODULE_PUBLIC_HEADERS )
+
+add_library(dyad SHARED ${DYAD_MODULE_SRC})
+set_target_properties(dyad PROPERTIES PREFIX "")
+target_link_libraries(dyad PUBLIC Jansson::Jansson flux::core)
+target_link_libraries(dyad PRIVATE dyad_dtl)
```

Comment:
In the `dtl` directory, the CMake target is named `${PROJECT_NAME}_dtl`. We should use the same here

Reply:
Fixed